### ISSUE TITLE
Issue on-screen warning if badly-formatted file is uploaded

### ISSUE THREAD

**Issue by c-simpson** [CodeForPhilly/paws-data-pipeline#342]
When watching the logs, you can see an    
 ` ERROR in file_uploader: No sources found in upload`    
message but a normal user has no way to know that the file was bad.

When user uploads a file that cannot be ingested, display a message like     
```Salesforce_PAWS Donation_9_1_2020_TO_3_30_2021.xlsx was not correctly formatted - make sure you are using the right report```
   
   

@kfettich - is there a better term than 'report' to describe the downloads from the PAWS standpoint?

**Comment by kfettich** [comment]
They are called 'reports' on the Salesforce UI and in the Volgistics UI, so this is probably a good term to use; I'd maybe rephrase the error message to `Salesforce_PAWS Donation_9_1_2020_TO_3_30_2021.xlsx columns do not match the expected formatting - are you uploading the right report?`

**Comment by c-simpson** [comment]
This should tie into #437 (Status Page)

**Comment by c-simpson** [comment]
If file is badly formatted, server returns '{False : "Similarity to expected column names below threshold"}'

**Comment by c-simpson** [comment]
See also https://github.com/CodeForPhilly/paws-data-pipeline/issues/336

**Comment by kfettich** [comment]
Bad formatting in shifts tab is unlikely (changes to Volgistics output are not known to happen). This issue is a nice to have, but not required.

**Comment by kfettich** [comment]
Out of scope
